tests/src/Functional/DeveloperDoesNotExistTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\apigee_edge\Functional;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class DeveloperDoesNotExistTest extends ApigeeEdgeFunctionalTestBase {
```

Comment:
Don't we have a test already that validates redirection to the api-communication-error page? If there is one then this should be merged with that. There is no need to setup Drupal twice to validate the same behavior twice.